tycho-core/src/main/java/org/eclipse/tycho/core/resolver/DefaultDependencyResolverFactory.java

```diff
@@ -47,18 +46,11 @@ public DependencyResolver lookupDependencyResolver(ReactorProject reactorProject
 
         Properties properties = reactorProject.getProperties();
         TargetPlatformConfiguration configuration = TychoProjectUtils.getTargetPlatformConfiguration(reactorProject);
-
         if (properties.getProperty("tycho.targetPlatform") != null) {
             throw new RuntimeException("-Dtycho.targetPlatform was deprecated and IS REMOVED in this tycho version.");
         }
         if (properties.getProperty("tycho.test.targetPlatform") != null) {
-            try {
-                @SuppressWarnings("deprecation")
-                String hint = org.eclipse.tycho.core.osgitools.targetplatform.LocalDependencyResolver.ROLE_HINT;
-                return container.lookup(DependencyResolver.class, hint);
-            } catch (ComponentLookupException e) {
-                throw new RuntimeException("Could not instantiate required component", e);
-            }
+            throw new RuntimeException("don't use that");
```

Comment:
@akurtakov Each test that currently fails with this message uses the old (or ancient) way of setting the TP and needs adjustments, so if you have an idea that would be good if you can help adjust them, or if they are no longer useful, delete them.